pom.xml

```diff
@@ -33,6 +35,7 @@
         <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+        <tag>${scmTag}</tag>
```

Comment:
@Vlatombe where does the `${scmTag}` variable come from?

Reply:
It is provided by incrementals infra. See https://github.com/jenkinsci/incrementals-tools